### DISCUSSION TITLE
Bookmark videos

### DISCUSSION THREAD

**Discussion by Ver-Nunft** [mediathekview/zapp#331] [Ideas]
Instead of downloading could we also have an option to bookmark a video?

**Comment by cemrich** [comment]
Yes, this is a feature currently in the making! See #143

**Reply by Ver-Nunft** [reply]
Well, it's taking its time 😬.
Is there an approximately ETA?

**Reply by cemrich** [reply]
Well, I'm a single developer with job, family and hobbies. So these things take time.

But here is a sneak peek at the new UI that will most likely be home to the latest downloads, viewing history and bookmarks:

![Screenshot_20221116-175641](https://user-images.githubusercontent.com/403202/202245519-8a211ac8-deb2-4fc7-9fce-fadbd618e5ff.jpg)

**Reply by Ver-Nunft** [reply]
Love what you are doing with the limited ressources available. I even donated some money for it to continue thriving. 

But what's the hold up for some kind of bookmarking? Couldn't it just be a rudimentary list at first?
